doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,215 @@
++----------------------------------+----------------------------------+----------------------------------+
+| Modulation in :term:`Uplink (UL)`| QPSK or 16QAM in both UL and DL  | BPSK or QPSK in UL depending on  |
+| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality.                  |
+|                                  |                                  | Only QPSK in DL                  |
++----------------------------------+----------------------------------+----------------------------------+
+
+Location support
+================
+
+A factor governing the selection of the access technology is the MNO support in the areas where you wish to deploy your product.
+Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
+See `Mobile IoT Deployment Map`_ for more information.
+
+Certifications
+==============
+
+Some MNOs, such as Verizon and Vodafone, have their own certification requirements.
+But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.
+
+For more information about certification related to the nRF91 Series SiP, refer to the following pages:
+
+* `nRF91 Series certifications`_
+* `Certifying a Cellular IoT Device`_
+
+SIM card support
+================
+
+SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT or both.
+The iBasis SIM is bundled with the nRF9160 DK and Thingy:91, and Onomondo SIM is bundled with the nRF9161 DK.
+Check the `iBasis IoT network coverage`_, `Onomondo LTE-M coverage`_, or `Onomondo NB-IoT coverage`_ pages to see the network coverage for different countries, depending on the SIM card you are using.
+
+Software SIM
+------------
+
+The nRF91 Series supports software SIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
+The following are some of the key advantages of using software SIM:
+
+* Cost savings and simplified design by excluding the SIM hardware layer.
+* Power savings especially in idle mode using eDRX.
+
+See the software SIM support section of the `Cellular IoT unique features`_ documentation for information on software SIM.
+
+The software SIM support is available in |NCS| from the v2.5.0 release.
+The `Onomondo SoftSIM integration with the nRF91 Series`_ guide describes the integration of software SIM into |NCS|, and the :ref:`nrf_modem_softsim` guide demonstrates the transfer of SIM data between the modem and the application.
+
+Power consumption
+=================
+
+LTE-M and NB-IoT both support Power Saving Mode (PSM) or Extended Discontinuous Reception (eDRX) to minimize power consumption.
+For more information on power-saving techniques, refer to the DevAcademy's `Power saving techniques`_ documentation.
+
+Security
```

Comment:
Would it make sense to mention https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/board_support/processing_environments.html and link to this page here?

Reply:
May be we could add after first sentence, where we mention about Arm TrustZone ? @tokangas : Could you please confirm.
Then, we are mainly talking about protecting data transmitted over the LTE network.

Reply:
Yes, it's fine by me.